src/time-range-picker/time-range-picker.tsx

```diff
@@ -184,7 +184,7 @@ export const TimeRangePicker = ({
                     onChange={handleStartTime}
                 />
 
-                <ArrowRangeIcon>
+                <ArrowRangeIcon tabIndex={-1}>
```

Comment:
if possible, remove the outer button and only render the icon

since this was mentioned in a [previous review comment](https://github.com/LifeSG/react-design-system/pull/133#discussion_r1190741040), check if this is already in progress to avoid conflicts 🙏